### PR TITLE
🎨 Name block redesign block redesign #624

### LINKED ISSUE #624: :art: Name Block Redesign: block redesign
## User stories

As a user, I want the "Name Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/name-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/name-message-block/`.

### PR DESCRIPTION
**Pull request by Mutagah**
# Description

This PR makes UI changes to name block redesign

Fixes #624

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Screenshot(optional)

name_block_1

name_block_2

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/name-message-block/src/lib/components/name-block/name-block.component.html [hunk 1]
```diff
@@ -3,7 +3,5 @@
 		<input [id]="nameInputId" formControlName="message" type="text"
 			[placeholder]="'PAGE-CONTENT.BLOCK.PLACEHOLDER.NAME' | transloco" />
 
-		<app-default-option-field [jsPlumb]="jsPlumb" [blockFormGroup]="nameMessageForm"></app-default-option-field>
-
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/name-message-block/src/lib/components/name-block/name-block.component.html:7]: We'll have to change the state layer to account for this, so for now just leave and then edit it to match the new design

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Same changes as those i've requested in #662

**Comment by LemmyMwaura** [comment]
Kindly Resolve your conflicts.

**Comment by Mutagah** [comment]
> Kindly Resolve your conflicts and leave the anchor as left for now @Mutagah

Changes requested made.

**Comment by LemmyMwaura** [comment]
Closing this as you've merged a lot of block design issues. So next step is to pick something a bit more challenging. I'll assign you an issue.

**Pull request closed without merging**